.github/workflows/periodic.yml

```diff
@@ -78,33 +78,34 @@ jobs:
       AWS_OSSCI_METRICS_V2_ACCESS_KEY_ID: ${{ secrets.AWS_OSSCI_METRICS_V2_ACCESS_KEY_ID }}
       AWS_OSSCI_METRICS_V2_SECRET_ACCESS_KEY: ${{ secrets.AWS_OSSCI_METRICS_V2_SECRET_ACCESS_KEY }}
 
-  linux-bionic-cuda10_2-py3_9-gcc7-build:
-    name: linux-bionic-cuda10.2-py3.9-gcc7
+  linux-bionic-cuda11_6-py3_7-gcc7-debug-build:
```

Comment:
Any story behind why we have debug build for 3.7 here but not 3.9 like others

Reply:
Hm, don't really know I think 1 debug build per python version is enough